README.md

```diff
@@ -1 +1,24 @@
-# lap3_code_challenge
```

Comment:
Really good README file. Clear and organised information and easy to understand.